### PR TITLE
Adding normalize to paddle frontend

### LINKED ISSUE #19696: normalize

### PR DESCRIPTION
**Pull request by Madjid-CH**
Close #19696

### CODE REVIEW

**Comment by Madjid-CH** [comment]
Hi @YushaArif99 
can please you review this PR and give me some feedbacks.
thanks

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @Madjid-CH 

The PR look good to me! Could you kindly fix the [linting errors](https://github.com/unifyai/ivy/actions/runs/5615841705/job/15216997969?pr=19716) showing up in the CI? Once those are fixed, we can then merge this PR. 
Thanks! 🙂

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Madjid-CH** [comment]
@YushaArif99 I reformatted the code
please review my changes.

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @Madjid-CH 

The linting failures are still being shown in the CI [here](https://github.com/unifyai/ivy/actions/runs/5678234732/job/15388048888?pr=19716). Could you kindly fix these.

Also, the CI tests for `test_paddle_normalize` are now failing due to a key error `cpu` ([link](https://github.com/unifyai/ivy/actions/runs/5845648474/job/15849955236?pr=19716)) Could you kindly resolve these as well. Thanks 🙂

**Comment by Madjid-CH** [comment]
Hi @YushaArif99, can you please look at my new changes.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py [hunk 1]
```diff
@@ -43,3 +43,38 @@ def test_paddle_layer_norm(
...
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        on_device=on_device,
+        fn_tree=fn_tree,
+        x=x[0],
+        p=p,
+        axis=0,
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_norm.py:79]: I just noticed this right now. But this is quite restrictive. `axis`. Could you kindly add value generation for the `axis` parameter as well following the domain defined in the paddle docs [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/normalize_en.html#:~:text=axis%20(int%2C%20optional)%20%E2%80%93%20The%20axis%20on%20which%20to%20apply%20normalization.%20If%20axis%20%3C%200%2C%20the%20dimension%20to%20normalization%20is%20x.ndim%20%2B%20axis.%20%2D1%20is%20the%20last%20dimension.)

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @Madjid-CH 

Apologies for the delay in the response. I was actually away for a couple of weeks due to which I couldn't respond to the PRs in time. Now that I am back, I'll make sure to be more responsive. 

BTW, could you kindly address the abovementioned points? I think once this is resolved and all the relevant tests still pass, the PR should be ready to be merged. Thanks! 🙂

**Comment by Madjid-CH** [comment]
Hey @YushaArif99 ,
would you please review my changes 😊.

**Review by YushaArif99 — APPROVED** [review]
Hi @Madjid-CH 

Thanks a lot for making the requested changes. The testing seems more robust now and all tests are also passing which is great to know. I'll be going ahead with the merge. Thanks for the contribution! 😄

**Pull request merged**